react/features/base/logging/middleware.js

```diff
@@ -99,11 +99,50 @@ function _initLogging(loggingConfig) {
  * specified {@code action}.
  */
 function _libWillInit({ getState }, next, action) {
-    _setLogLevels(JitsiMeetJS, getState()['features/base/logging'].config);
+    // Adding the if in order to preserve the original logic after enabling
+    // LIB_WILL_INIT for the web version.
+    if (typeof APP === 'undefined') {
```

Comment:
Is the if-check needed around _setLogLevels? I can understand wrapping "_setErrorHandlers" in an if because it seems fairly web specific. I don't think it's worth preserving old logic if it doesn't need to be preserved (but maybe it does, you have more context).

Reply:
I'm not sure if it is safe to remove the `if`.

Reply:
It's worth trying and seeing what errors. That is probably a more useful comment than "Adding the if in order to preserve the original logic..." because some day someone will be like "what original logic? why can't we remove the if?"

I'll approve the PR so you can merge it if you need but I think it's something you should try.